code/modules/mob/living/living.dm

```diff
@@ -751,6 +746,8 @@ default behaviour is:
 			add_overlay(A)
 
 /mob/living/Destroy()
+	clear_inventory_slots()
+	. = ..()
```

Comment:
Why are you calling the parent's Destroy() prior to removing auras? Shouldn't almost all Destroy() behavior on non-top-level types be done prior to the parent call?